node/tests/ServerModules.test.ts

```diff
@@ -1418,6 +1425,385 @@ describe("Server Module Tests", () => {
+                    await GlideJson.set(
+                        client,
+                        prefixBicycles + 8,
+                        ".",
+                        '{"brand": "nHill", "model": "Summit", "price": 1200, "condition": "new"}',
+                    ),
+                ).toEqual("OK");
+
+                expect(
+                    await GlideJson.set(
+                        client,
+                        prefixBicycles + 9,
+                        ".",
+                        '{"model": "ThrillCycle", "brand": "BikeShind", "price": 815, "condition": "refurbished"}',
+                    ),
+                ).toEqual("OK");
+
+                // let server digest the data and update index
+                await new Promise((resolve) =>
+                    setTimeout(resolve, DATA_PROCESSING_TIMEOUT),
+                );
+
+                // FT.AGGREGATE idx:bicycle * LOAD 1 __key GROUPBY 1 @condition REDUCE COUNT 0 AS bicycles
+                const options: FtAggregateOptions = {
+                    loadFields: ["__key"],
+                    clauses: [
+                        {
+                            type: "GROUPBY",
+                            properties: ["@condition"],
+                            reducers: [
+                                {
+                                    function: "COUNT",
+                                    args: [],
+                                    name: "bicycles",
+                                },
+                            ],
+                        },
+                    ],
+                };
+                const aggreg = await GlideFt.aggregate(
+                    client,
+                    indexBicycles,
+                    query,
+                    options,
+                );
+                const expectedAggreg = [
+                    {
+                        condition: "new",
+                        bicycles: isResp3 ? 5 : "5",
+                    },
+                    {
+                        condition: "refurbished",
+                        bicycles: isResp3 ? 1 : "1",
+                    },
+                    {
+                        condition: "used",
+                        bicycles: isResp3 ? 4 : "4",
+                    },
+                ];
+                expect(
+                    aggreg
+                        .map(convertGlideRecordToRecord)
+                        // elements (records in array) could be reordered
+                        .sort((a, b) =>
+                            a["condition"]! > b["condition"]! ? 1 : -1,
+                        ),
+                ).toEqual(expectedAggreg);
+
+                const aggregProfile = (
+                    await GlideFt.profileAggregate(
+                        client,
+                        indexBicycles,
+                        "*",
+                        options,
+                    )
+                )[0] as FtAggregateReturnType;
```

Comment:
You can remove cast if you update return type
```suggestion
                )[0];
```